<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1680</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1680</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="SavingsSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>530</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>装饰</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/up.png</normaloff>:/res/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>171</width>
      <height>91</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="SavingsRead">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>存款读取</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/up.png</normaloff>:/res/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>171</width>
      <height>91</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="WithdrawlRead">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>530</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>取款读取</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/up.png</normaloff>:/res/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>171</width>
      <height>91</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>731</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sortName">
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sortType">
          <property name="text">
           <string>储种</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sortCapital">
          <property name="text">
           <string>本金</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sortDate">
          <property name="text">
           <string>到息日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sortLost">
          <property name="text">
           <string>是否挂失</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>挂失日期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>营业员姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>营业员工号</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listSavings">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWithdrawls">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>130</y>
      <width>821</width>
      <height>349</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>  工号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>  姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="staffLineID">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="staffLineName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>当前工作人员：</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="changeStaff">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>更换工作人员</string>
    </property>
   </widget>
   <widget class="QToolButton" name="designateStaff">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>指名工作人员</string>
    </property>
   </widget>
   <widget class="QToolButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>记录此序列</string>
    </property>
   </widget>
   <widget class="QToolButton" name="backButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>回退记录序列</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>702</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sortName_2">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sortType_2">
       <property name="text">
        <string>储种</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sortCapital_2">
       <property name="text">
        <string>本金</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sortDate_2">
       <property name="text">
        <string>到息日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>是否挂失</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>营业员姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>营业员工号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string> 取款日期</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>播放音乐</string>
    </property>
   </widget>
   <widget class="QToolButton" name="SavingsSave_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>530</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>装饰</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/up.png</normaloff>:/res/up.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>171</width>
      <height>91</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>新建</string>
    </property>
    <addaction name="actionAddSaving"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionLostHandler"/>
    <addaction name="actionWithdrawlHandler"/>
    <addaction name="AllSave"/>
    <addaction name="AllSaveByUser"/>
    <addaction name="AllSaveLast"/>
    <addaction name="actionPrint"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddSaving">
   <property name="text">
    <string>存款</string>
   </property>
  </action>
  <action name="actionWithdraw">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>取款</string>
   </property>
  </action>
  <action name="actionLostHandler">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>挂失处理</string>
   </property>
  </action>
  <action name="actionWithdrawlHandler">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>办理取款</string>
   </property>
  </action>
  <action name="AllSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存(默认)</string>
   </property>
  </action>
  <action name="AllSaveByUser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="AllSaveLast">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存(上一次)</string>
   </property>
  </action>
  <action name="actionSaveWithdrwals">
   <property name="text">
    <string>SaveWithdrwals</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打印利息单</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>查询到期</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
